app/src/main/java/org/mozilla/focus/screenshot/ScreenshotViewerActivity.java

```diff
@@ -69,11 +69,11 @@
     private static final int ACTION_SHARE = ACTION_VIEW + 2;
     private static final int ACTION_DELETE = ACTION_VIEW + 3;
 
-    private static int REQUEST_CODE_VIEW_SCREENSHOT = 101;
```

Comment:
These REQUEST_CODE_VIEW_SCREENSHOT are actually not used any more in a recent patch, I should have removed that :P but yeah that's another issue.